course_gather/viewsets.py

```diff
@@ -48,6 +50,11 @@ class CourseViewSet(viewsets.ModelViewSet):
     filterset_class = CourseFilter
     filter_backends = (filters.DjangoFilterBackend,)
 
+    # Cache for 5 minutes
```

Comment:
We should look into caching at the db or memory level instead, rather than just caching a single viewset.

Reply:
Do you mean changing the cache backend from the default filesystem backer to the [database backend](https://docs.djangoproject.com/en/3.0/topics/cache/#database-caching)?

Django also supports Memcached, but that seems overkill at this point.

Reply:
Yep, the database backend. I agree that memcached is a little overkill for now. Maybe if the project ever gets bigger

Reply:
967fec1. There seems to be a slight performance penalty of ~100 ms when using database caching over filesystem caching on my machine (NVME drive, 6 core CPU) but that shouldn't be too big of a issue.

Reply:
I think this penalty should be okay. If we really want to get _max_ performance we should be using memcached